### PR TITLE
Tres novas ativididades adicionadas

### PR DESCRIPTION
**Pull request by kakomarina**

### CODE REVIEW

File: pages/activities/atividade4.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      <div className="flex justify-center items-center bg-fixed bg-cover bg-center container-top shadow-md">
+        <h1 className="text-4xl text-center text-white">Activities</h1>
+      </div>
+      <div className="container mx-auto px-4 py-8">
+        <div className="flex bg-white my-4 rounded-lg shadow-md">
+          <div className="py-2 pr-2 xl:py-4 xl:pr-4">
+            <h2 className="news-title sm:text-lg">
+              Pint of Science 2018
+            </h2>
+            <h3 className="news-date text-xs pb-2 xl:pb-4">05/12/1999</h3>
```
Comment (Es7evam) [inline pages/activities/atividade4.js]: Please fix the dates :)

File: pages/activities/atividade5.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      <div className="flex justify-center items-center bg-fixed bg-cover bg-center container-top shadow-md">
+        <h1 className="text-4xl text-center text-white">Activities</h1>
+      </div>
+      <div className="container mx-auto px-4 py-8">
+        <div className="flex bg-white my-4 rounded-lg shadow-md">
+          <div className="py-2 pr-2 xl:py-4 xl:pr-4">
+            <h2 className="news-title sm:text-lg">
+              Regional School of Informatics Workshop
+            </h2>
+            <h3 className="news-date text-xs pb-2 xl:pb-4">05/12/1999</h3>
```
Comment (Es7evam) [inline pages/activities/atividade5.js]: another date to be fixed

File: pages/activities/atividade6.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      <div className="flex justify-center items-center bg-fixed bg-cover bg-center container-top shadow-md">
+        <h1 className="text-4xl text-center text-white">Activities</h1>
+      </div>
+      <div className="container mx-auto px-4 py-8">
+        <div className="flex bg-white my-4 rounded-lg shadow-md">
+          <div className="py-2 pr-2 xl:py-4 xl:pr-4">
+            <h2 className="news-title sm:text-lg">
+              Pint of Science 2018
+            </h2>
+            <h3 className="news-date text-xs pb-2 xl:pb-4">05/12/1999</h3>
```
Comment (Es7evam) [inline pages/activities/atividade6.js]: the last date fix

**Review by Es7evam — CHANGES_REQUESTED** [review]
Please fix the event dates

File: pages/activities/atividade5.js (hunk 2 above)
Comment (Chaves2021) [reply pages/activities/atividade5.js]: Date fixed

File: pages/activities/atividade6.js (hunk 3 above)
Comment (Chaves2021) [reply pages/activities/atividade6.js]: Date fixed

**Pull request merged**